src/Domain/Value/QueryParameter/QueryParameterCollection.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of storyblok/php-content-api-client.
+ *
+ * (c) Storyblok GmbH <info@storyblok.com>
+ * in cooperation with SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Storyblok\Api\Domain\Value\QueryParameter;
+
+/**
+ * @author Frank Stelzer <dev@frankstelzer.de>
+ * @author Silas Joisten <silasjoisten@proton.me>
```

Comment:
@silasjoisten as I have copied some functionality I left you here as author. Please let me now, if this is OK or if I should remove it.

Reply:
i think we dont need this class in general.

Reply:
moved directly to StoriesRequest, without Collection.